src/renderer/features/albums/components/album-list-content.tsx

```diff
@@ -104,6 +110,14 @@ export const AlbumListView = ({
         };
     }, [query, overrideQuery]);
 
+    const columns = useMemo(() => {
+        if (showRatings) {
+            return table.columns;
+        } else {
+            return table.columns.filter((column) => column.id !== 'userRating');
+        }
+    }, [table, showRatings]);
```

Comment:
I would remove this change from all table column instances, as the columns are already configurable to remove the rating column if not desired.

Reply:
I reverted all the changes on the table columns. I ended up having to change how the cards choose to display the rating stars as well. My initial approach was to remove the onRating completely from all controls but this causes weirdness when the user had the ratings hidden on the settings but choose to display the ratings on any of the tables because the ratings would show up and be interactive on hover but clicking on it would have no effect and I found that interaction to be weird.